### DISCUSSION TITLE
Using !inner with or

### DISCUSSION THREAD

**Discussion by chrisLoPresti** [supabase/supabase#5899] [Questions]
Is there anyway to use !inner with or? 

For example: select('\*\, product!inner(*)').or(`product.name.ilike.%${VALUE}%, otherValue.ilike..%${VALUE}%, .... `)

Currently this does not work for me

**Comment by GaryAustin1** [comment]
Two issues:
First
I don't believe .or works with !inner but has a foreign table option. (I could be wrong on this but see 2nd link)
https://supabase.com/docs/reference/javascript/or#use-or-on-foreign-tables
issues discussed here:
https://github.com/PostgREST/postgrest/issues/2011
Second
Independently .or requires all filter sets to be in same table.
